distribution/receiver/routes/receiver-route.xml

```diff
@@ -91,12 +91,18 @@
                     <to uri="jpa:ReceiverSyncRequest" />
 
                     <log message="Successfully saved updates for sync request" loggingLevel="DEBUG" />
+                    
+                    <setProperty name="receiverSyncRequest">
```

Comment:
Move the code so that you store the sync request in the property before saving it to the database